app/src/main/java/org/septa/android/app/services/apiinterfaces/Favorites.java

```diff
@@ -12,14 +14,30 @@
 
 public interface Favorites {
 
-    public Map<String, Favorite> getFavorites(Context context);
+    Map<String, Favorite> getFavorites(Context context);
 
-    public void addFavorites(Context context, Favorite favorite);
+    List<FavoriteState> getFavoriteStates(Context context);
 
-    public void deleteFavorite(Context context, String id);
+    void addFavorites(Context context, Favorite favorite);
 
-    public void deleteAllFavorites(Context context);
+    void addFavoriteState(Context context, FavoriteState favoriteState);
```

Comment:
Instead of an addFavoritesState maybe setFavoriteStates and take a List<FavoriteState>.  Part of that state is going to be order.